viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -84,6 +84,7 @@ import { Image360ApiHelper } from '../../api-helpers/Image360ApiHelper';
 import html2canvas from 'html2canvas';
 import { AsyncSequencer, SequencerFunction } from '../../../../utilities/src/AsyncSequencer';
 import { getNormalizedPixelCoordinates } from '@reveal/utilities';
+import { FlexibleCameraManager } from '@reveal/camera-manager/src/Flexible/FlexibleCameraManager';
```

Comment:
Add `FlexibleCameraManager` to `packages/camera-manager/index.ts`, and simplify this import path to just `@reveal/camera-manager`